R/dataProcess.R

```diff
@@ -125,6 +129,79 @@ dataProcess = function(
     output
 }
 
+#' Feature-level data summarization with multiple cores
+#' 
+#' @param input feature-level data processed by dataProcess subfunctions
+#' @param method summarization method: "linear" or "TMP" 
+#' @param equal_variance only for summaryMethod = "linear". Default is TRUE. 
+#' Logical variable for whether the model should account for heterogeneous variation 
+#' among intensities from different features. Default is TRUE, which assume equal
+#' variance among intensities from features. FALSE means that we cannot assume 
+#' equal variance among intensities from features, then we will account for
+#' heterogeneous variation from different features.
+#' @param censored_symbol Missing values are censored or at random. 
+#' 'NA' (default) assumes that all 'NA's in 'Intensity' column are censored. 
+#' '0' uses zero intensities as censored intensity. 
+#' In this case, NA intensities are missing at random. 
+#' The output from Skyline should use '0'. 
+#' Null assumes that all NA intensites are randomly missing.
+#' @param remove50missing only for summaryMethod = "TMP". TRUE removes the runs 
+#' which have more than 50\% missing values. FALSE is default.
+#' @param impute only for summaryMethod = "TMP" and censoredInt = 'NA' or '0'. 
+#' TRUE (default) imputes 'NA' or '0' (depending on censoredInt option) by Accelated failure model. 
+#' FALSE uses the values assigned by cutoffCensored
+#' @param numberOfCores Number of cores for parallel processing. When > 1, 
+#' a logfile named `MSstats_dataProcess_log_progress.log` is created to 
+#' track progress. Only works for Linux & Mac OS. Default is 1.
+#' 
+#' @importFrom parallel makeCluster parLapply stopCluster clusterExport  
+#' 
+#' @return list of length one with run-level data.
+#' 
+MSstatsSummarizeV2 = function(input, method, impute, censored_symbol,
```

Comment:
MSstatsSummarize

Reply:
I take it back, a different name will be better

Reply:
Maybe `MSstatsSummarizeWithMultipleCores`?

Reply:
that works, thanks